include/CISolver/CISolver.hpp

```diff
@@ -0,0 +1,52 @@
+#ifndef GQCG_CISOLVER_HPP
+#define GQCG_CISOLVER_HPP
+
+
+#include <HamiltonianBuilder/HamiltonianBuilder.hpp>
+#include <HamiltonianParameters/HamiltonianParameters.hpp>
+#include <WaveFunction.hpp>
+#include <numopt.hpp>
+
+namespace GQCG {
+
+
+/**
+ *  Class which requires a HamiltonianBuilder and HamiltonianParameters
+ *  so that it can find a set of eigenvalues and -vectors for the Hamiltonian (which cannot always be stored in memory)
```

Comment:
Put something in the lines of 'solves the CI eigenvalue problem' earlier in the documentation